hack/generated/controllers/generic_controller.go

```diff
@@ -530,21 +550,17 @@ func (gr *GenericReconciler) ManageOwnership(ctx context.Context, action Reconci
 			return ctrl.Result{}, errors.Wrap(err, "patching resource error")
 		}
 
-		return ctrl.Result{
-			// TODO: We should consider a scaling backoff here, see: https://github.com/Azure/k8s-infra/issues/263
```

Comment:
Removed this comment: controller-runtime already implements a scaling backoff, see: https://stuartleeks.com/posts/error-back-off-with-controller-runtime/